databricks/koalas/frame.py

```diff
@@ -9370,7 +9410,7 @@ def op(kser):
         return self._apply_series_op(op)
 
     # TODO: axis = 1
-    def idxmax(self, axis=0):
+    def idxmax(self, axis=0) -> Union["ks.Series", "DataFrame", "ks.Index"]:
```

Comment:
Sounds good! In this case, I have to mark its return statement as `# type: ignore`, otherwise
```
databricks/koalas/frame.py: note: In member "idxmax" of class "DataFrame":
databricks/koalas/frame.py:9488: error: Incompatible return value type (got "Union[Series[Any], DataFrame[Any], Index]", expected "Series[Any]")
```